### PR TITLE
Add style guide

### PR DESCRIPTION
**Pull request by mathiasayivor**
## 👨‍💻 Changes proposed

Added Prettier, Editorconfig and Eslint as discussed in #15 and re-introduced in #48 It should be left with creating a STYLE_GUIDE.md file, but that shouldn't be a priority now since the packages added here solves most of the problem already.

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

* Some new packages were added to client (check client/package.json)
* New files were added to client directory
* Changes that appeared in most most of the files are just reformatting, done by prettier and eslint.

### CODE REVIEW

File: client/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        }
+    },
+    "env": {
+        "browser": true,
+        "jest": true,
+        "node": true
+    },
+    "rules": {
+        "array-bracket-newline": "error",
+        "camelcase": "error",
```
Comment (Dun-sin) [inline client/.eslintrc.json:22]: why no camelCase?

Comment (mathiasayivor) [reply client/.eslintrc.json:22]: It actually means only camelCasing is accepted...

Comment (Dun-sin) [reply client/.eslintrc.json:22]: alright got it

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Efficient

**Comment by Dun-sin** [comment]
you should fix the merge conflict

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable

**Comment by mathiasayivor** [comment]
merge conflict resolved

**Comment by Dun-sin** [comment]
Thank you so much for contributing to this project and following the contributing rules, this looks good and will be merged.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient, Elegant, Maintainable

**Pull request merged**